Source/santad/EventProviders/SNTEndpointSecurityClientTest.mm

```diff
@@ -481,6 +487,7 @@ - (void)testProcessMessageHandler {
 
   auto mockESApi = std::make_shared<MockEndpointSecurityAPI>();
   mockESApi->SetExpectationsRetainReleaseMessage();
+  ::testing::Mock::AllowLeak(mockESApi.get());
```

Comment:
Note: Test method has a large comment as to why this leak is expected and benign.